Makefile

```
@@ -66,16 +73,19 @@ build-host-local-plugin:
 	fi
 
-test: $(GO) build-host-local-plugin
+unit-tests: $(GO)
 	$(GO) test -mod=readonly ./cmd/... ./pkg/... -v --ginkgo.v
 
-docker-test:
-	hack/test-dockerized.sh
-
-test-%: $(GO) build-host-local-plugin
-	$(GO) test ./$(subst -,/,$*)/... -v --ginkgo.v
+cni-tests: $(GO)
```

checkmake (0.3.2)

[warning] Target body for "cni-tests" exceeds allowed length of 5 lines (7).

(maxbodylength)